src/definitions.ts

```diff
@@ -78,26 +82,30 @@ export interface GoogleLoginOptions {
    */
   nonce?: string;
   /**
-   * Set if your application needs to refresh access tokens when the user is not present at the browser.
-   * In response use `serverAuthCode` key
+   * Set if your application requires to force the refreshToken [Android only]
    *
    * @default false
    * @since 3.1.0
    * */
-  grantOfflineAccess?: boolean;
+  forceRefreshToken?: boolean;
 }
 
-export interface GoogleLoginResponse {
+export interface GoogleLoginOnlineResponse {
+  responseType: "online";
   accessToken: AccessToken | null;
-  idToken: string | null;
```

Comment:
This is not what you and I agreed on. 
![image](https://github.com/user-attachments/assets/1acdcb0f-9f95-424a-9375-78e9b5df5ccf)

This entire PR is not JUST about offline mode. It also introduces the "fixes" that unify the experience between all 3 platforms. As you probably remember well, it's not possible to provide `idToken` on the web, so we BOTH decided that the best course of action would be to just remove it entirely.

This PR is meant to be breaking. It should receive a new major release.

Reply:
Correct, bit right now idtoken is used by some users who migrate and use only native, and it’s also common in all auth we have so. I believe maybe we can make it as a config option to use the new system ? And in major it will be by default , or with option to disable it ?

Reply:
Well, the problem is going to be explaining to users that `idtoken` is native only. I am like 99% sure that you will get users complaining about `idtoken` not working on the web and explaining to people that it's an unsolvable issue is not great. I sort of think we should be consistent in this. This development branch has been up for a couple of weeks at this point and I have already instructed users how to migrate AWAY from `idtoken`.

Personally, I do not believe that it is that big of a deal. If you make it a breaking change with a new major, then it's sort of normal that a migration might be necessary.

Reply:
I believe that verifying `idtoken` server-side is what Google docs recommend: https://developers.google.com/identity/gsi/web/guides/verify-google-id-token#node.js

Reply:
I am well aware, however as I already said, google does not provide a modern SDK to get the `idtoken` on the web.

To be more specific, the provide a way to render a [Sign in with google button](https://developers.google.com/identity/gsi/web/guides/overview)
![image](https://github.com/user-attachments/assets/49657daf-d1ba-4261-bee1-96aad92515c5)
This button does in fact return `idtoken`.

However, this requires the end user to make changes to their UI for the web version. I do not believe that this is acceptable, and as such `idtoken` will be removed from ALL the platforms, including iOS and Android.

Providing a unified user experience between all the platforms is the highest priority of this plugin.
I will reconsider adding `idtoken` when Google updates theirs SDKs